themes/src/main/resources/theme/base/admin/resources/partials/realm-identity-provider-github-ext.html

```diff
@@ -0,0 +1,18 @@
+<fieldset>
```

Comment:
That is using existing styling for collapsed for that advanced override section. I can change default to have it not collapsed if you prefer. There are a several tooltip messages that this PR includes in admin-messages_en.properties that this file uses that suggest that default value is used unless an override value is provided.

Reply:
@vmuzikar do you want me to change default to have this expanded vs collapsed for these advanced settings?

Reply:
I wouldn't use override section for this at all. IMHO it can be placed directly among other config options. Look e.g. at PayPal. It has a sandbox settings (which effectively changes the endpoints) and it's not placed in a collapsed section.

If we place in in the collapsed override section, that section should probably be placed at the very bottom, similarly to e.g. client config.

Reply:
I believe the UI issue is still valid.